libs/zzz/db/src/Database/DataManagers/CharacterOptManager.ts

```diff
@@ -273,15 +299,36 @@ export class CharacterOptManager extends DataManager<
           }) as BonusStatTag,
           value,
         }
-      }
-    )
+      })
+      .filter(notEmpty)
 
     if (!critModeKeys.includes(critMode)) critMode = 'avg'
 
     if (typeof enemyLvl !== 'number') enemyLvl = 80
     if (typeof enemyDef !== 'number') enemyDef = 953
     enemyisStunned = !!enemyisStunned
 
+    if (typeof enemyStunMultiplier !== 'number') enemyStunMultiplier = 150
```

Comment:
Why store percent as whole number instead of 1.5? Seems inconsistent with how we handle percents

Reply:
actually all values are stored as percents, and converted to decimal upon being put into calc.
![image](https://github.com/user-attachments/assets/e11c788e-9157-4aeb-a864-7c78118fd698)
It is consistent with how we handle percents.